src/main/java/ari/Ari.java

```diff
@@ -0,0 +1,52 @@
+package ari;
+
+import ari.command.ByeCommand;
+import ari.command.Command;
+import ari.parser.Parser;
+import ari.storage.Storage;
+import ari.tasks.TaskList;
+import ari.ui.Ui;
+
+/**
+ * Represents Ari functions
+ */
+public class Ari {
+    private String filePath;
+
+    private Storage store;
+    private TaskList toDoList;
+    private Ui ui;
+
+    /**
+     * Constructor of Ari
+     *
+     * @param path path to location of save file
+     */
+    public Ari(String path) {
+        filePath = path;
+        ui = new Ui();
+    }
+
+    /**
+     * Starts the program
+     */
+    public void run() {
+        ui.displayWelcomeMessage();
+
+        Parser par = new Parser();
```

Comment:
```suggestion
        Parser parser = new Parser();
```
Consider changing the naming variable to match the convention. Since the Ui variable is named ui and the Storage variable is named storage.